### DISCUSSION TITLE
EM7455 in a M.2 to PCIe adapter with SIM slot for desktop

### DISCUSSION THREAD

**Discussion by anishjp** [danielewood/sierra-wireless-modems#141] [Q&A]
Hi Guys

I have a EM7455 in a Dell laptop working perfectly.

I was wondering if I could use the EM7455 in a "M.2 to PCIe" adapter with a SIM slot (like this one http://www.hwtools.net/Adapter/PN12A.html ) and use it in a desktop?

Has anyone tried this? Will it work? Any advise?

Regards,
Anish

**Comment by danielewood** [comment]
Will likely work fine. You **may** have to use kapton tape on the PCIe connector, as PCIe signals could cause problems the the EM74xx, but probably not an issue. The 7455 operates entirely over USB, so you only need to make sure you have that working.

**Reply by anishjp** [reply]
Many thanks Daniel.

I know you mostly work with Sierra modems, but do you probably know, if I use a modem like Fibocom L850-GL (that works over PCIe only) in an adapter, will I still have to mask any pins?
